.github/workflows/ci-pipeline-arm64.yml

```diff
@@ -39,12 +39,15 @@ jobs:
       fail-fast: false
       matrix:
         cc: [gcc, clang]
-        os: ['ubuntu_16.04', 'ubuntu_18.04']
+        os: ['ubuntu_16.04', 'ubuntu_18.04', rocky_linux_8]
```

Comment:
`rocky_linux_8` could be in quotes for symmetry.

Reply:
Fixed.